exercises/practice/sgf-parsing/.docs/instructions.md

```diff
@@ -0,0 +1,66 @@
+# Description
+
+Parsing a Smart Game Format string.
+
+[SGF](https://en.wikipedia.org/wiki/Smart_Game_Format) is a standard format for
+storing board game files, in particular go.
+
+SGF is a fairly simple format. An SGF file usually contains a single
+tree of nodes where each node is a property list. The property list
+contains key value pairs, each key can only occur once but may have
+multiple values.
+
+An SGF file may look like this:
+
+```text
+(;FF[4]C[root]SZ[19];B[aa];W[ab])
+```
+
+This is a tree with three nodes:
+
+- The top level node has three properties: FF\[4\] (key = "FF", value
+  = "4"), C\[root\](key = "C", value = "root") and SZ\[19\] (key =
+  "SZ", value = "19"). (FF indicates the version of SGF, C is a
+  comment and SZ is the size of the board.)
+  - The top level node has a single child which has a single property:
+    B\[aa\].  (Black plays on the point encoded as "aa", which is the
+    1-1 point (which is a stupid place to play)).
```

Comment:
Is this really in the problem specs instructions?

```suggestion
    B\[aa\].  (Black plays on the point encoded as "aa", which is the
    1-1 point).
```

Reply:
https://github.com/exercism/problem-specifications/pull/1814

Reply:
I personally find it pretty amusing :)

Reply:
Ya, but we've made a conscious effort to remove this type of comment/test case from problems.

Reply:
Yes, that's probably the right decision. What do I do for now? Remove it here and you do a PR? Or do we wait for your PR?
Is there a reason why the specifications are duplicated in the tracks repositories? I would think that a central one might work best, with appendices added by the tracks as necessary, which is kind of what's happening now anyway.

Reply:
Problem specs will take time to approve, and afaik it doesn't sync yet.  I think there is a copy for the same reason as right now it doesn't pull the instructions centrally.  I think this is future work to be done, just hasn't been prioritized ahead of other product work for v3.